### PR TITLE
Cheaper IBC deposit fees

### PR DESCRIPTION
**Pull request by assafmo**
Reduced fees on IBC Deposit from Secret, Cosmos & Terra by setting more appropriate gas limits.

### CODE REVIEW

**Comment by ValarDragon** [comment]
err, how were these gas constants obtained? They are oddly, vastly different

**Comment by assafmo** [comment]
Os Secret we have a lower gas setup, on other chains from observations. Why was 500k decided for other chains? Why is it 1.35M on Osmosis?

**Comment by ValarDragon** [comment]
Oh, odd. I didn't notice it was 1.35M vs 500k earlier, my bad!

File: src/stores/root.ts [hunk 1]
```diff
@@ -98,13 +98,18 @@ export class RootStore {
...
 				},
 				chainOpts: this.chainStore.chainInfos.map(chainInfo => {
+					let gas = 500_000;
+					if (chainInfo.chainId.startsWith('osmosis-')) {
+						gas = 1_350_000;
+					} else if (chainInfo.chainId.startsWith('secret-')) {
+						gas = 25_000;
+					} else if (chainInfo.chainId.startsWith('columbus-') || chainInfo.chainId.startsWith('cosmoshub-')) {
+						gas = 100_000;
+					}
```
Comment (ValarDragon) [inline src/stores/root.ts:108]: Can we revert this clause? Then looks good to merge to me

Comment (assafmo) [reply src/stores/root.ts]: Yes, done. :+1:

**Review by ValarDragon — APPROVED** [review]
LGTM!

**Pull request merged**